ember-cli-build.js

```diff
@@ -17,12 +17,14 @@ module.exports = function (defaults) {
     }
   })
 
-  /*
-    This build file specifes the options for the dummy test app of this
-    addon, located in `/tests/dummy`
-    This build file does *not* influence how the addon or the app using it
-    behave. You most likely want to be modifying `./index.js` or app's build file
-  */
+  if (app.env === 'test') {
```

Comment:
Mainly b/c we don't rely on those for what this provides, I only used them to test the code provided here.